MediaAdminBundle/Transformer/FolderTransformer.php

```diff
@@ -40,6 +64,23 @@ public function transform($folder)
     }
 
     /**
+     * @param FacadeInterface $facade
+     * @param mixed|null      $source
+     *
+     * @return mixed
+     */
+    public function reverseTransform(FacadeInterface $facade, $source = null)
+    {
+        if ($source instanceof MediaFolderInterface) {
+            $parent = $this->folderRepository->findOneById($facade->parentId);
+            $source->setParent($parent);
+            $source->setPath($parent->getPath() . '/' . $source->getFolderId());
```

Comment:
As subscriber changes the sons paths, i would prefer to only change parent here and dispatch a parent_updated event and change all paths in subscriber.